web-console/4.0-designs/storage/object-storage-dashboard/object-storage-dashboard.md

```diff
@@ -0,0 +1,91 @@
+**Only used resources types will be shown, no need to show types that are not in used. 
+
+## Health Card
+![Health and alerts](img/health-card.png)
+
+The health indication shows if the system is up and running without any major issues. When the system is in a warning or error state the alerts section will be shown and the relevant alerts will be displayed there. 
+
+## Data Consumption Card
+In this card the user can see an analysis for providers and accounts (MCG accounts) by different KPIs (physical Usage, logical usage, i/o, and egress depends on the selected type). 
+
+### Accounts> I/O operations
+![Accounts by I/O operations](img/data-consumption-crad-01.png)
+
+Show the top 5 accounts consumers R/W count values. At the bottom legends show the total Reads and total Writes amount of all 5. 
+Why? States how connected applications utilizes the system resources. 
+
+### Accounts > Logical Usage
+![Accounts by Logical Usage](img/data-consumption-crad-02.png)
+
+Show each account total logical usage size
+Why? Show how connected applications utilizes the system resources. 
+
+### Providers> I/O operations
+![Providers by I/O operations](img/data-consumption-crad-04.png)
+
+Input/Output operations sent to and from MCG to connected cloud providers
+Why? What’s eating my budget? Understanding the traffic in the cloud, allowing the user to compare with the cloud billing reports, optimize and improve resources allocation according to the I/O pattern and perhaps consider changing the selected resource type.
+
+### Providers> Egress
+![Providers by Egress](img/data-consumption-crad-05.png)
+
+The size of the data NooBaa read from each provider 
+Why? Cost analysis.  Understanding the traffic in the cloud, allowing the user to compare with the cloud billing reports, optimize and improve resources allocation according to the egress pattern and perhaps consider changing the selected resource type.
+    
+### Providers> Physical vs. logical usage
+![Providers by Physical vs. logical usage](img/data-consumption-crad-03.png)
+
+Show the data consumption in the system by comparing the raw usage (physical) vs. the logical usage. Raw usage refers to the actual size that the buckets in the system are utilizing from it's resources including data resiliency replicas or fragments. 
+Why? Allows the user to control storage efficiency and allocate appropriate resources according to usage characteristics.
+
+## Data Resiliency Card
+![Data Resiliency](img/data-resiliency-crad.png)
+
+In this card we will show if there is any resiliency issue in MCG/Noobaa. For any rebuilding process we will show a progress bar that indicates the progression. Time estimation will be shown as well if available.
+
+## Capacity Usage Card
+![Capacity Usage Breakdown](img/capacity-usage-crad.png)
+Show a breakdown by pie chart of the usage of the object storage by projects/ Bucket Class.
```

Comment:
I think an extra space is needed before this line. Looks like the text is starting right after the image in the preview.